### PR TITLE
create edit delete user

### PR DESCRIPTION
**Pull request by vuducluan**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -3,4 +3,48 @@ class Admin::UsersController < ApplicationController
...
   def index
   end
+
+  def create
+    if @user.save
+      flash[:success] = t "user.create_success"
+      redirect_to [:admin, @user]
+    else
+      flash[:warning] = t "user.create_failed"
+      redirect_to new_admin_user_url
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: render :new

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -3,4 +3,48 @@ class Admin::UsersController < ApplicationController
...
+  def edit
+  end
+
+  def update
+    if @user.update_attributes user_params
+      flash[:success] = t "user.update_success"
+      redirect_to [:admin, @user]
+    else
+      flash[:warning] = t "user.update_failed"
+      redirect_to edit_admin_user_path @user
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: render :edit

**Comment by hellovietnam93** [comment]
OK

File: app/views/admin/users/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  <%= f.text_field :email, class: "form-control" %>
+  <%= f.label :chatworkid %>
+  <%= f.text_field :chatworkid, class: "form-control" %>
+  <%= f.label :password %>
+  <%= f.password_field :password, class: "form-control" %>
+  <%= f.label :password_confirmation %>
+  <%= f.password_field :password_confirmation, class: "form-control" %>
+  <%= f.label :role %>
+  <%= f.select :role, User.roles.keys.to_a %>
+  <%= f.submit t("button.submit"), class: "btn btn-default btn-large btn-primary" %>
```
Comment (lyxuansang91) [inline app/views/admin/users/_form.html.erb]: 80 characters

**Pull request merged**